### PR TITLE
feat(dino-park): adjusting staff icon in profile page

### PR DESCRIPTION
**Pull request by andrew-sunada**
Adjusting staff icon location according to jira story.

Jira: https://jira.mozilla.com/browse/DP-1417

### CODE REVIEW

**Comment by fiji-flo** [comment]
I'd love to get Megan's input here.

**Comment by mbransn** [comment]
Thanks for the comment here and in Jira, have responded there & will connect with Andrew on this during our mtg tomorrow.

**Comment by andrew-sunada** [comment]
I had to adjust the Icon component to take a dynamic viewBox for the SVG because the staff icon didn't look correct according to the mockups.
@fiji-flo

File: src/components/ui/Icon.vue [hunk 1]
```diff
@@ -455,12 +455,21 @@
 </template>
 
 <script>
+const viewBoxValues = {
+  staff: '5 5 14 14',
```
Comment (fiji-flo) [inline src/components/ui/Icon.vue]: image
This breaks the original use cases for the staff icon.

File: src/pages/PageOrgchart.vue [hunk 2]
```diff
@@ -81,14 +84,21 @@ const fetcher = new Fetcher({ failoverOn: [302] });
...
   const e = document.createElement('div');
   const hasChildren = node.children.length > 0;
+  const IconClass = Vue.extend(Icon);
+  const icon = new IconClass({
+    propsData: {
+      id: 'staff',
+      width: 10,
+      height: 10,
+    },
+  }).$mount();
```
Comment (fiji-flo) [inline src/pages/PageOrgchart.vue]: Not super critical but this is a lot of overhead in the orgchart.

**Comment by LeoMcA** [comment]
We should probably take this opportunity to use proper svg sprites instead of inlining all our svgs multiple times: https://jira.mozilla.com/browse/IAM-230

This also makes reusing the "S" even simpler, as we can use a `<use href` within another `<use href`, like so:

```
<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
  
  <symbol id="fish" viewBox="0 26 100 48">
    <path d="M98.5,47.5C96.5,45.5,81,35,62,35c-2.1,0-4.2,0.1-6.2,0.3L39,26c0,4.5,1.3,9,2.4,12.1C31.7,40.7,23.3,44,16,44L0,34
	c0,8,4,16,4,16s-4,8-4,16l16-10c7.3,0,15.7,3.3,25.4,5.9C40.3,65,39,69.5,39,74l16.8-9.3c2,0.2,4.1,0.3,6.2,0.3
	c19,0,34.5-10.5,36.5-12.5S100.5,49.5,98.5,47.5z M85.5,50c-1.4,0-2.5-1.1-2.5-2.5s1.1-2.5,2.5-2.5s2.5,1.1,2.5,2.5
	C88,48.9,86.9,50,85.5,50z"/>  
  </symbol>
  
  <symbol id="fish-in-box" viewBox="0 0 120 68">
    <rect width="120" height="68" style="fill:red;" />
    <use href="#fish" width="100" height="48" x="10" y="10" />
  </symbol>
  
</svg>

<svg width="100"><use href="#fish"/></svg>

<svg width="100"><use href="#fish-in-box" /></svg>
```
Screenshot from 2019-08-21 15-24-16

Live demo here: https://jsfiddle.net/5r6zgvjq/

**Comment by andrew-sunada** [comment]
> We should probably take this opportunity to use proper svg sprites instead of inlining all our svgs multiple times: https://jira.mozilla.com/browse/IAM-230
> 
> This also makes reusing the "S" even simpler, as we can use a `<use href` within another `<use href`, like so:
> 
> ```
> <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
>   
>   <symbol id="fish" viewBox="0 26 100 48">
>     <path d="M98.5,47.5C96.5,45.5,81,35,62,35c-2.1,0-4.2,0.1-6.2,0.3L39,26c0,4.5,1.3,9,2.4,12.1C31.7,40.7,23.3,44,16,44L0,34
> 	c0,8,4,16,4,16s-4,8-4,16l16-10c7.3,0,15.7,3.3,25.4,5.9C40.3,65,39,69.5,39,74l16.8-9.3c2,0.2,4.1,0.3,6.2,0.3
> 	c19,0,34.5-10.5,36.5-12.5S100.5,49.5,98.5,47.5z M85.5,50c-1.4,0-2.5-1.1-2.5-2.5s1.1-2.5,2.5-2.5s2.5,1.1,2.5,2.5
> 	C88,48.9,86.9,50,85.5,50z"/>  
>   </symbol>
>   
>   <symbol id="fish-in-box" viewBox="0 0 120 68">
>     <rect width="120" height="68" style="fill:red;" />
>     <use href="#fish" width="100" height="48" x="10" y="10" />
>   </symbol>
>   
> </svg>
> 
> <svg width="100"><use href="#fish"/></svg>
> 
> <svg width="100"><use href="#fish-in-box" /></svg>
> ```
> 
> Screenshot from 2019-08-21 15-24-16
> 
> Live demo here: https://jsfiddle.net/5r6zgvjq/

I don't disagree with you. I've created an IAM story for this: https://jira.mozilla.com/browse/IAM-234

File: src/components/ui/Icon.vue (hunk 1 above)
Comment (andrew-sunada) [reply src/components/ui/Icon.vue]: I've made an adjustment here for that. Good catch!

**Comment by andrew-sunada** [comment]
Yes, I intentionally made that change, thinking that's where we should go, but I also didn't check with @mbransn . I can change it to look like nothing changed.

**Comment by LeoMcA** [comment]
Now that we display an "m" on staff profile pictures, rather than the S, I don't think is necessary.

**Pull request closed without merging**